src/lib/modules/guilds/types.ts

```diff
@@ -0,0 +1,29 @@
+import type { RecordModel } from "pocketbase";
+import type { User } from "../authentication";
+
+
+
+export interface Guild extends RecordModel {
+    name : string,
+    descriptions : string,
+    owner_id : User,
+    status : string,
+}
+
+
+
+export enum GuildStatusDTO  {
+    APPROVED = 'approved',
+    DEACTIVATE = 'deactivate',
+    REJECT = 'reject',
+    CLOSED = 'closed',
+    PENDING = 'pending'
+}
```

Comment:
```suggestion
export type GuildStatusValues = typeof GuildStatus[keyof typeof GuildStatus]
export const GuildStatus =  {
    APPROVED: 'approved',
    DEACTIVATE: 'deactivate',
    REJECT: 'reject',
    CLOSED: 'closed',
    PENDING: 'pending'
} as const
```

Reply:
Hello @arielrecto , let's avoid using enums, and use const POJOs together with type instead for performance reasons. thanks.